.github/workflows/copy-md.yml

```diff
@@ -26,25 +31,19 @@ jobs:
 
       - name: Install Dependencies
         run: |
-          yarn install --frozen-lockfile  
+          yarn install --frozen-lockfile
 
       - name: Run Sync Script
         run: |
-          chmod +x sync_script.sh  
+          chmod +x sync_script.sh
           ./sync_script.sh
 
-      - uses: stefanzweifel/git-auto-commit-action@v4
-        if: always()
+      - name: Add And Commit
+        uses: EndBug/add-and-commit@v9
         with:
-          commit_message: sync documentation changes
-
-      - name: Push Changes
-        run: git push origin sync-docs-branch
+          message: 'Updated MD File Changes from cosmos-sdk/docs'
 
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v3
+      - name: Push To Sync-Docs-Branch
+        uses: ad-m/github-push-action@master
         with:
-          base: main
-          branch: sync-docs-branch
-          title: "Sync Documentation Changes"
-          body: "Automatically generated by GitHub Actions to sync documentation changes."
+          branch: sync-docs-branch
```

Comment:
Why is it not creating a PR?
Or are you going to have another action to automate that part?

Reply:
I tried 3 auto PR actions and they kept resetting all the changes. Ive been trying this since friday and over the weekend and couldnt get it to work

Reply:
what is it failing on?

Reply:
it looks like there are no changes when this action runs. checked the last few runs. I think its cause main was updated recently. Lets merge this and then adding a pr action I can do in a follow up